sessions/recap-project-6-NEW/user-stories-completed/activity-planner/2_activity-details.md

```diff
@@ -0,0 +1,42 @@
+# Activity Details
```

Comment:
are these completed US here for reference or only during the review process. I think we should remove them since it would be easy for the students to find them and copy the content.